src/translations/nbTranslations.json

```diff
@@ -532,6 +532,7 @@
       "institution_admin": "Administrator",
       "institution_admin_description": "Vedlikeholder institusjonens opplysninger i personregisteret og tildeler roller i tjenesten.",
       "institution_admins": "Administratorer",
+      "no_active_employments": "Du har ingen roller fordi du har ingen aktive ansettelser og har ikke tilgang til å registrere.",
```

Comment:
Litt usikker på om det skulle vært "...og har derfor ikke tilgang til å registrere"

Reply:
Tenker det holder med "Du har ingen roller da du ikke har noen aktive ansettelser." 
Men det burde gått innom innom Cecilie/Terje for avklaring

Reply:
gjerne list opp de andre feilmeldingene vi har her også, så vi får et grønt lys på alle